pkg/skippeers/skippeers.go

```diff
@@ -139,25 +111,18 @@ func (l *List) prune() {
 // Must be called under lock
 func (l *List) pruneChunk(ch string, now int64) int {
 
-	for l.minHeap.Len() > 0 && l.minHeap.Peek() <= now {
-		heap.Pop(l.minHeap)
-	}
-
 	count := 0
 
-	chunkPeers := l.skip[ch]
-	chunkPeersLen := len(chunkPeers)
-	for peer, exp := range chunkPeers {
-		if exp <= now {
-			delete(chunkPeers, peer)
-			count++
-			chunkPeersLen--
+	if chunkPeers, ok := l.skip[ch]; ok {
```

Comment:
The if statement is unnecessary here. The `l.skip[ch]` can be used in the for loop directly.